core/domain/classifier_domain.py

```diff
@@ -277,6 +277,27 @@ def status(self):
     def training_data(self):
         return self._training_data
 
+    def update_state_name(self, state_name):
```

Comment:
No, if we're not using it, let's remove it now. We don't want dead code in the codebase. If we need it we can reinstate it later.